config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics.yaml

```diff
@@ -13,6 +13,10 @@ periodics:
       repo: cluster-api-provider-azure
       base_ref: master
       path_alias: "sigs.k8s.io/cluster-api-provider-azure"
+    - org: kubernetes
```

Comment:
added the k8s repo ref to the periodic job as well based on @detiber's review comment